src/sdk/Audio/AudioConfig.ts

```diff
@@ -114,6 +114,27 @@ export abstract class AudioConfig {
         return new AudioOutputConfigImpl(new SpeakerAudioDestination());
     }
 
+    /**
+     * Creates an AudioConfig object representing the custom IPlayer object.
+     * You can use the IPlayer object to control pause, resume, etc.
+     * @member AudioConfig.fromSpeakerOutput
+     * @function
+     * @public
+     * @param {IPlayer} player - the IPlayer object for playback.
+     * @returns {AudioConfig} The audio output configuration being created.
+     * Added in version 1.12.0
+     */
+    public static fromSpeakerOutput(player?: IPlayer): AudioConfig {
```

Comment:
>IPlayer [](start = 45, length = 7)

Do we need to export this if it's a type parameter.

Reply:
I added some doc and export it.